pkg/tsdb/prometheus/time_series_query_test.go

```diff
@@ -631,16 +631,16 @@ func TestPrometheus_parseTimeSeriesResponse(t *testing.T) {
 			End:          time.Unix(4, 0).UTC(),
 			UtcOffsetSec: 0,
 		}
-		res, err := parseTimeSeriesResponse(value, query, true)
+		res, err := parseTimeSeriesResponse(value, query)
```

Comment:
Do we still need this test as we are not null-filling anymore and the purpose of this test was to find out if nulls were inserted correctly?

Reply:
@ivanahuckova i switched the test to check that nulls are not inserted when there are gaps in the data. think that's worth keeping, or should i remove it?